code/parse/sexp.cpp

```diff
@@ -26817,6 +26835,10 @@ int query_operator_argument_type(int op, int argnum)
 			else
 				return OPF_SHIP_WING;
 
+		case OP_TIME_TO_GOAL:
+			if (argnum == 0)
```

Comment:
Flagged by CI. This causes a fall-through case when argnum != 0. Briefly looking at the other cases doesn't give me a clear idea of whats supposed to happen here, obviously something needs to be returned as the else statement, but I don't know what that should be at this time.

Reply:
You're returning the parameter type for each parameter.  But this sexp only has *one* parameter, so there is no need to specifically check for the first one.  Just remove the if() here and always return `OPF_SHIP`.